File: fstp-core-bo/src/main/java/com/purefun/fstp/example/bo/ExampleQnsBO.java
package com.purefun.fstp.example.bo;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

import org.apache.ignite.cache.query.annotations.QuerySqlField;

import com.purefun.fstp.core.tool.UUID;
import com.purefun.fstp.core.tool.fstpbo;

@Entity  
@Table(name="ExampleQns") 
@fstpbo(boid=6015L,destination = "fstp.example.bo.test")
public class ExampleQnsBO extends ExampleBO  {					
	@QuerySqlField
	public String company = "";
}

File: fstp-python/com/purefun/fstp/example/bo/model/ExampleQnsBO.py
import uuid

class ExampleQnsBO(object):

    def __init__(self):
        self.company = ''
        self.name = ''
        self.age = 0
        self.uuid = str(uuid.uuid1())
        self.boid = 6015
        self.destination = "fstp.example.bo.test"
